cloud/home/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Home | Terramate Cloud
+description: Learn about the Home page of the Terramate Cloud dashboard
+---
+
+# Home Page
+
+The Home page provides a summary of your engagements in Terramate Cloud, including:
+
+- **Alerts**: Open alerts assigned to you.
+- **Pull Requests**: Open pull requests authored by you.
+- **Deployments**: Deployments in which you collaborated.
```

Comment:
For deployments, maybe we can use the same title as the table above: `Deployments you collaborated on.`

Only a suggestion from my side.

Reply:
thanks @SavanovicN , I've applied this suggestion to change the headings of the sections which follows this section. In this section, the point is to inform the reader about the key things which they can read in the Home screen.